contents/blog/product-engineer-tools.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2022-12-14
+title: The best tools for product engineers
```

Comment:
Given there isn't much search volume specifically around tools for product engineers atm, I think we could use a more shareable headline that still works for SEO. Something like...

- x tools product engineers swear by
- The X tools product engineers can't live without
- The essential tools for product-minded engineers

Thoughts?

Reply:
The 6 essential tools of product engineers?